fintech-examples/fintech-server/src/main/resources/application.yml

```diff
@@ -25,7 +25,6 @@ spring:
     driver-class-name: org.h2.Driver
 
 fintech-ui:
-  ok-url: http://localhost:5500/fintech-callback/redirect?fintechRedirectUriOk={redirectCode}
```

Comment:
mock:
  tppais:
    listtransactions: true
    listaccounts: true
should be set to false, please.